<launch>
    <node pkg="grid_2d_map" type="pcdtogridmap_node" name="pcdtogridmap" output="screen" />
    <!-- <arg name="file_directory" default="filtered_map" />
    <arg name="file_name" default="/home/qjs/" /> -->
    <arg name="z_adjust" default="0.27" /> -->

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find grid_2d_map)/config/rviz_test.rviz" />
    </group>


</launch>
